nifi-ngsild-postgresql-processors/src/main/java/egm/io/nifi/processors/ngsild/utils/NGSIUtils.java

```diff
@@ -150,6 +150,13 @@ private Attribute parseNgsiLdAttribute(String key, JSONObject value, boolean fla
                 "parametername", "Property", "", "", "", "", key.toLowerCase(), false, null
             );
             subAttributes.add(parameterName);
+            if (Objects.equals(datasetId, "")){
+                datasetId = "default";
+            }
+            Attribute parameterDatasetId = new Attribute(
+                "datasetid", "Property", "", "", "", "", datasetId.toLowerCase(), false, null
```

Comment:
toLowerCase() can make two differents datasetIds similar after an export (for example if we have Dataset:Raw and Dataset:raw). I don't really know if that is actually an issue, I just did the same as what had been done for parameter_name ...

Reply:
well, you should avoid such subtle differences in namings, so error prone. so that's OK for me.